src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -12,22 +12,42 @@ Set up your Centrifuge Mainnet or Testnet node.
 - [Releases](https://github.com/centrifuge/centrifuge-chain/releases)
 - Docker OR `rustup` is installed
 
-## Steps
+### Optional: Using a snapshot from the Centrifuge dev team
```

Comment:
Could you provide some more context around this? Do you do this in addition to running with Docker / running with binaries, or is it something that allows you to skip that?

Reply:
Added context. This is optional, it will pre-populate the blockchain DB instead of starting from scratch whether you use Docker or not it doesn't matter. But it has to be done before starting up any services.